package.json

```diff
@@ -60,6 +60,8 @@
     "dotenv": "^6.2.0",
     "express": "^4.16.4",
     "express-validator": "^5.3.1",
```

Comment:
Did you forget to remove `express-validator`?

Reply:
@deschant

Reply:
Yes I did, removing it now